plugin/bufsize/README.md

```diff
@@ -16,6 +16,7 @@ The default value is 512, and the value must be within 512 - 4096.
 Only one argument is acceptable, and it covers both IPv4 and IPv6.
 
 ## Examples
+### As a forwarder
```

Comment:
just drop this subsection and add the usecase to the prose. Also I think it makes sense to say you want a larger bufsize instead of the 512 default, but then also warn about fragmentation, maybe (again?)

Reply:
Thank you for reviewing!
Ok, I'll drop subsection and use prose style.
Also, I agree with mentioning about larger bufsize consideration. And I think it would be better to mention it on **Considerations** section.